src/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs/promises');
+const { sep, basename } = require('path');
+
+const fileParam = process.argv[2];
+const endpointParam = process.argv[3];
+
+moveFile(fileParam, endpointParam);
+
+async function moveFile(file, endpoint) {
+  try {
+    const dataOfFile = await fs.readFile(file);
+
+    await fs.writeFile(endpoint, dataOfFile);
+    await fs.rm(file);
+  } catch (err) {
+    if (err.code === 'EROFS') {
```

Comment:
The code should throw an error if the destination directory does not exist, instead of attempting to move the file to the current directory.